docs/solana/migration/blockchain-node.mdx

```diff
@@ -49,8 +52,26 @@ $ solana-keygen pubkey /var/data/13cQ6QUDvebHAppdmLYu9KXJvNZS9voVQCn8XrMbBemiLJ3
 6ucMsEXv7KLW9V6dfGYAsx5PF4yy2hAB79nURwTuuzgN
 ```
 
-5. Verify your token balance on devnet using a public explorer. Solana Explorer listed below.
+5. Get the list of serialized transactions, which should be sent to Solana:
+
+```
+$ http -b https://migration.web.helium.io/migrate/13cQ6QUDvebHAppdmLYu9KXJvNZS9voVQCn8XrMbBemiLJ3r8Z5?limit=1000&offset=0
+
+```
+
+6. Submit the transactions to Solana. You can see typescript code demonstrating how to use this
+[here](https://github.com/helium/helium-program-library/blob/master/packages/migration-service/src/test-submit.ts#L37).
+
+Alternatively, you can use the [Migration Widget](https://docs.helium.com/solana/migration/exchange#mapping-helium-wallets-to-solana-wallets)
+or [command line utility](https://github.com/helium/helium-program-library/releases/tag/v0.0.27-migration-cli)
+for steps 5-6. Example command provided below for the command line utility:
+
+```
+$ env MIGRATION_SERVICE_URL=https://migration.web.helium.io env SOLANA_URL=https:/api.mainnet-beta.solana.com env SOLANA_WSS_URL=wss://api.mainnet-beta.solana.com ./migration-tx-executor --wallet 6ucMsEXv7KLW9V6dfGYAsx5PF4yy2hAB79nURwTuuzgN
+```
+
+7. Verify your token balance on devnet using a public explorer. Solana Explorer listed below.
 
 ```
-https://explorer.solana.com/address/$SOLANA_ADDRESS?cluster=devnet
+https://explorer.solana.com/address/$SOLANA_ADDRESS
```

Comment:
an example could be useful here to help distinguish whether the `$` is part of the placeholder or the URL

Reply:
Good catch